services/dynamic_addressbook_service.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+option java_package = "com.hederahashgraph.service.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.token">>> This comment is special code for setting PBJ Compiler java package
+
+import "query.proto";
+import "response.proto";
+import "transaction_response.proto";
+import "transaction.proto";
+
+/**
+ * Transactions for Dynamic Address Book Service
+ */
+service DynamicAddressBookService {
```

Comment:
I don't think we should have anything related to `DynamicAddressBook` in this PR. I think we should name it `NodeService`.

a) Dynamic is redundant as everything in an API is by its definition dynamic.
b) Our API is designed around the individual "entities" in our system like accounts, contracts, etc. So by that measure it should be named as `Node` something. This PR has it named after a grouping of nodes. It's like instead of a `FileService` we named it `FolderService` or `FileSystemService`.
c) We already have a `NodeStakeUpdateTransaction` and we should group it under the same prefix.
d) We already have a MAPI `/api/v1/network/nodes` and `getNodes` so we should be consistent with mirror node.

Reply:
Changed to NodeService.